cloudhub-2/modules/ROOT/pages/_partials/create-config.adoc

```diff
@@ -25,9 +25,11 @@ the allowed block size that you can assign for your Anypoint VPC is between `/16
 
 When specifying CIDR blocks, ensure that the CIDR blocks:
 
-* Are from a private IP space
-* Don't overlap with any other CIDR blocks assigned to your other private spaces
-* Don't overlap with any CIDR blocks in use in your corporate network
+* Are from a private IP space.
+* Don't overlap with any other CIDR blocks assigned to your other private spaces. When you enter a CIDR block during private space creation, CloudHub 2.0 checks the value to ensure it does not overlap with another CIDR that is currently in use.
```

Comment:
I'm not sure we need to make changes here, the CH2 check to ensure not no CIDR overlapping is for reserved CIDRs only, not for CIDR in general.